src/roles/webfilter-deploy/tasks/main.yml

```diff
@@ -56,14 +56,21 @@
       path: /tmp/webfilter
       state: absent
 
-  - name: "Ensure application properties contains webfilter"
+  - name: "Adding the {{ cert_name }} jks to application.properties"
```

Comment:
This will fail if the cert_name is undefined. Looks like the code has this.  
{% if item.cert_name is defined %}
cert_name: "{{ item.cert_name }}"
{% endif %}
we should either default it there as webfilter or default it in this file. 
Also please run a test with cert_name not defined (single webfiler) and make sure it works.